algorithms/scripts/WIP-passiveaggressive-KT12.py

```diff
@@ -0,0 +1,310 @@
+
+# coding: utf-8
+
+# # Create a passive aggressive classifier to predict TP53 mutation from gene expression data in TCGA
+
+# In[1]:
+
+import os
+import urllib
+import random
+import warnings
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+from sklearn import preprocessing, grid_search
+from sklearn.cross_validation import train_test_split
```

Comment:
See imports from [`2.TCGA-MLexample.ipynb`](https://github.com/cognoma/machine-learning/blob/7406e51f087932062a076dfbd5573c0d0fd8cbc2/2.TCGA-MLexample.ipynb) for how to rid these warnings.